mim/commands/install.py

```diff
@@ -244,7 +244,7 @@ def patched_iter_dependencies(self, extras=()):
         if self.canonical_name not in PKG2PROJECT or self.canonical_name == 'mmcv-full':  # noqa: E501
             return deps
 
-        if 'mim' in self.iter_provided_extras:
+        if 'mim' in str(self.iter_provided_extras):
```

Comment:
```suggestion
        if 'mim' in self.iter_provided_extras():
```

Reply:
Hi @danielhonies , thanks for your contribution. In fact, `self.iter_provided_extras` is a method so we can call this method rather than using str.